src/App.js

```diff
@@ -14,286 +14,431 @@ export default App;
 
 function SelectColumn(props) {
   return (
-    <button className="selectColumn-button" onClick={props.onClick}>
-
-    </button >
-  )
+    <button className="selectColumn-button" onClick={props.onClick}></button>
+  );
 }
 
 function DisplayHoles(props) {
   return (
     <div className={props.className}>
       <h1>{props.value}</h1>
     </div>
-  )
-};
+  );
+}
 
 class Game extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      col1: [],
-      col2: [],
-      col3: [],
-      col4: [],
-      col5: [],
-      col6: [],
-      col7: [],
+      columns: [[], [], [], [], [], [], []],
+      // board: [
```

Comment:
@jasmine-ship-it  - I left the other potential way that state can be modelled, commented, for reference!